src/gsy_e/models/strategy/scm/load.py

```diff
@@ -29,11 +30,30 @@ def _get_day_of_timestamp(self, time_slot: DateTime) -> int:
 
     def market_cycle(self, area: "AreaBase") -> None:
         """Update the load forecast and measurements for the next/previous market slot."""
+        self._energy_params.add_entry_in_hrs_per_day(area._current_market_time_slot)
+        self._update_energy_requirement_and_measurement(area)
+
+    def _update_energy_requirement_and_measurement(self, area: "AreaBase"):
+        self._energy_params.update_energy_requirement(area._current_market_time_slot)
+
+        if not self._energy_params.allowed_operating_hours(area._current_market_time_slot):
```

Comment:
Well, this specific method (allowed_operating_hours) is still needed because in the former discussion we decided to maintain the hrs_of_day parameter and to remove support for the hrs_per_day, however I reviewed the code again and you are right, by mistake I added the "EnergyParameters" class that includes the hrs_per_day instead of the one that does not! Adapting the SCM strategy to use the simpler interface, thanks again for bringing this up!